src/lib_windows.cr

```diff
@@ -105,6 +109,40 @@ lib LibWindows
 
   @[CallConvention("X86_StdCall")]
   fun wsa_startup = WSAStartup(version : Int16, data : WSAData*) : Int32
+
+  # source https://msdn.microsoft.com/en-us/library/windows/desktop/ms724950(v=vs.85).aspx
+  struct SystemTime
+    year : Word
+    month : Word
+    day_of_week : Word
+    day : Word
+    hour : Word
+    minute : Word
+    second : Word
+    milliseconds : Word
+  end
+
+  # source https://msdn.microsoft.com/en-us/library/windows/desktop/ms725481(v=vs.85).aspx
+  struct TimeZoneInformation
+    bias : Long
+    standard_name : StaticArray(WChar, 32)
+    standard_date : SystemTime
+    standard_bias : Long
+    daylight_name : StaticArray(WChar, 32)
+    daylight_date : SystemTime
+    daylight_bias : Long
+  end
+
+  struct FileTime
+    low_date_time : DWord
+    high_date_time : DWord
+  end
```

Comment:
this way it matches docs. UInt64 would work since it will be guaranteed to be aligned at it is used as out in the method call. 

In the docs it is warned the following thing:

> Do not cast a pointer to a FILETIME structure to either a ULARGE_INTEGER* or __int64* value because it can cause alignment faults on 64-bit Windows.

If `struct FileTime` to UInt64 there will be no problem since it is the cast is in the other way, but it is misleading.

What do you prefer?